cxplat/src/cxplat_winkernel/memory_winkernel.c

```diff
@@ -56,3 +57,40 @@ cxplat_free(_Frees_ptr_opt_ void* pointer, cxplat_pool_flags_t pool_flags, uint3
         ExFreePoolWithTag(pointer, tag);
     }
 }
+
+_Must_inspect_result_ cxplat_status_t
+cxplat_allocate_lookaside_list(
```

Comment:
since winuser and winkernel both seem to call "initialize" (not "allocate") type APIs internally, should this be named `cxplat_initialize_lookaside_list`?